src/main/pages/workspace-admin/projects.adoc

```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
+The following shows example projects:
 
-[id="into"]
-== Into
+image::workspaces/projects.png[]
 
-You can step into or out of the project tree. If you step into a folder, that folder will be set as the project tree root and the explorer will redraw itself. All commands are then executed against this folder root.
 
 [id="project-type-definition"]
-== Project Type Definition
+== Project type definition
 
 Plug-in developers can define their own project types. Since project types trigger certain behaviors within the IDE, the construction of the projects is important to understand.
 
-1.  *A project has type.* Project type is defined as one primary type and zero or more mixin types. A primary project type is one where the project is editable, buildable and runnable. A mixin project type defines additional restrictions and behaviors of the project, but by itself cannot be a primary project type. The collection of primary and mixin types for a single project define the aggregate set of attributes that will be stored as meta data within the project.
-2.  *Project types describe different aspects of a project* such as types of source files inside, the structure of the explorer tree, the way in which a command will be executed, associated workflows, and which plug-ins must be installed.
-3.  *A project defines a set of attributes.* The attributes of a project can be mandatory or optional. Attributes that are optional can be dynamically set at runtime or during configuration.
-4.  *Sub-projects may have different project types than their parents.* Modules may physically exist within the tree structure of the parent (as its subfolders) or outside (the parent is a soft link to the module project).
+1.  A project has a type.  The project type is defined as one primary type and zero or more mixin types. A primary project type is one where the project is editable, buildable, and runnable. A mixin project type defines additional restrictions and behaviors of the project, but by itself cannot be a primary project type. The collection of primary and mixin types for a single project define the aggregate set of attributes that will be stored as meta data within the project.
+2.  Project types describe different aspects of a project such as types of source files inside, the structure of the explorer tree, the way in which a command will be executed, associated workflows, and which plug-ins must be installed.
+3.  A project defines a set of attributes. The attributes of a project can be mandatory or optional. Attributes that are optional can be dynamically set at runtime or during configuration.
+4.  Sub-projects may have different project types than their parents. Modules may physically exist within the tree structure of the parent. For example, subfolders exists within the tree structure of the parent.   Also, modules may exist outside such as when the parent is a soft link to the module project.
```

Comment:
"For example, subdirectories exist within the tree structure of the parent."

Also, there is excess whitespace (three spaces?) before "Also, modules may".

By "outside", do you mean outside of the tree structure? I'm not sure what this means.